A <%= @exception.class %> occurred in <%= @job.class.to_s %>:

  <%= @exception.message %>
  <%= @backtrace.first %>

  <%= @params.inspect %>

  User : <%= User.find(@job.user_id).inspect %>
  Project : <%= Project.find(@job.project).inspect %>